src/components/README.md

```diff
@@ -12,10 +12,13 @@ information.
 | ----------------- | ------------------------------------------------------------------------------------ |
 | document          | any (json data from [our hook](../hooks/README.md#usePlatformBridgeDocument))        |
 | componentRegistry | Map<string, Config<any>> from [@measuredco/puck](https://github.com/measuredco/puck) |
+| themeConfig?      | ThemeConfig                                                                          |
 
 ### Usage:
 
 ```tsx
+import { themeConfig } from "../../theme.config"
```

Comment:
think this import for the README is wrong? it'd be like from "@yext/visual-editor" ?

Reply:
This is the import of the developer's theme.config.ts, so it is a local path in the repo.
That being said, the theme features will need a lot better documentation, I just didn't want to write too much until the tailwind stuff Brian is doing is set.

Reply:
Oh gotcha